mobile-app/lib/service/authentication_service.dart

```diff
@@ -126,13 +125,12 @@ class AuthenticationService {
     bool authRes = await extractCookies();
 
     if (!authRes) {
-      var navRes = await Navigator.push(
```

Comment:
Currently I'm returning a message saying the authentication was a success which I think we can use in future(for failing cases) so could you add this variable back.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix